apisix/cli/file.lua

```diff
@@ -234,6 +237,20 @@ function _M.read_yaml_conf(apisix_home)
         end
     end
 
+    if default_conf.apisix.config_center == "yaml" then
```

Comment:
Can we put this in apisix/core/config_yaml.lua?

Reply:
At first I wanted to put this in apisix/core/config_yaml.lua, but because the file loads ngx variables, it needs to be loaded in the openresty environment. Can't run in CLI mode, so we can't put this in apisix/core/config_yaml.lua